src/python/pants/backend/debian/target_types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.goals.package import OutputPathField
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    DictStringToStringField,
+    SpecialCasedDependencies,
+    StringField,
+    Target,
+)
+from pants.util.docutil import doc_url
+
+
+class DebianControlFile(StringField):
+    alias = "control"
+    help = "Path to a Debian control file for the package to be produced."
+
+
+class DebianSymlinks(DictStringToStringField):
+    alias = "symlinks"
+    help = "Symlinks to create for each target being packaged."
+
+
+class DebianInstallPrefix(StringField):
+    alias = "install_prefix"
+    default = "/opt"
+    help = "Absolute path to a directory where Debian package will be installed to."
```

Comment:
Is the idea Pants will install into the system for you? I don't think we want that, Pants generally should not be writing outside of your build root (except for the cache directories, which can be changed). 

Pants should only create the artifact and put it into `dist/`, then you manually install it.

Reply:
Oh sorry if my help string wasn't clear! One needs an installation prefix for a Debian package to specify where the Debian package will be installed on the system (once the packaged is produced). This information determines the directory hierarchy within the package. 

```
├── example-pex
│   ├── DEBIAN
│   │   └── control
│   ├── opt
│   │   └── company
│   │       └── example
│   │           └── example-linux-cp3.6.pex
│   └── usr
│       └── bin
│           └── do-work -> /opt/company/example/example-linux-cp3.6.pex
```

Here the user has specified /opt/company/example as installation prefix and thus a Debian package with these directory tree is produced. When user will install the Debian package, the files will end up under their parent directories on the file system (so the `example-linux-cp3.6.pex` will end up under `/opt/company/example` directory.

So Pants of course will not install the Debian package -- the installation prefix is needed solely to produce a Debian with the right directory hierarchy. Hope this makes sense now?

Reply:
Great! That makes a lot more sense, thank you for explaining. 

This is probably overly chatty, but perhaps this would make it more clear for newbies like me? 

> Absolute path to a directory where the Debian package will be installed to when installing the `.deb` file (e.g. via `dpkg`).

Edit: Actually, probably fine to keep as is if you update the `help` for the target as suggested below.